### ISSUE TITLE
Create email comfirmed UI

### ISSUE THREAD

**Issue by abha224** [anitab-org/open-source-programs-web#29]
## Description
As a user, after successful registration, my email has to be confirmed. I need to see an email confirmation message in osp once verification is done.

### Update [Required]
- [ ] Create email confirmed UI
- [ ] must have an appropriate message of confirmation.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentors.

## Estimation
4 hrs

**Comment by m-code12** [comment]
@abha224 Is this issue still open?

**Comment by NarimaneHennouni** [comment]
Hi, is this still open?

**Comment by emmanuelhashy** [comment]
Hi, abha224 I'd like to work on this issue.

**Comment by vj-codes** [comment]
@emmanuelhashy hey are you still interested to work on this issue?

**Comment by Rahulm2310** [comment]
@vj-codes I would like to take up this issue.

**Comment by aakankshaagr** [comment]
@vj-codes If this issue is available I would like to work on this issue.

**Comment by emmanuelhashy** [comment]
No

**Comment by mayburgos** [comment]
> @vj-codes If this issue is available I would like to work on this issue.

You were assigned!

**Comment by emmanuelhashy** [comment]
No

**Comment by subodhk01** [comment]
Is someone working on this issue? I would love to help

**Comment by Rahulm2310** [comment]
@subodhk01 Thanks for showing interest but it is already been assigned to @aakankshaagr . Feel free to look through some other available issues.
@mayburgos Can you please remove the `Available` label as it might be confusing for a contributor. Thanks 🙂

**Comment by Aaishpra** [comment]
> @vj-codes If this issue is available I would like to work on this issue.

What are your updates @aakankshaagr are you still working on this or are you stuck somewhere do you need help? please update us in a maximum of 2 days or this issue will be made available for others

**Comment by aakankshaagr** [comment]
@Aaishpra  I am still working on this issue. I am working for the first time on such issues so it's taking me longer. I am sorry for the delay. I think by this week this issue will be solved. I will surely update my progress after every 2 days.

**Comment by Aaishpra** [comment]
No problem @aakankshaagr i just wanted to remind you about the issue, it's good that you are still interested in the issue and workinh on it. Let us know if you face any doubts 😉

**Comment by aakankshaagr** [comment]
@Aaishpra  Earlier I forgot to ask before starting working on this issue. I saw that in swagger UI required verification and confirmation messages are present for user registration. Only email confirmation of account verification is absent. So, I have created a template for account verification successfully. 
So what I have done is correct or not?
If not then can anyone help me by explaining this issue?

**Comment by Aaishpra** [comment]
> @Aaishpra  Earlier I forgot to ask before starting working on this issue. I saw that in swagger UI required verification and confirmation messages are present for user registration. Only email confirmation of account verification is absent. So, I have created a template for account verification successfully. 
> So what I have done is correct or not?
> If not then can anyone help me by explaining this issue?

I believe it is correct as the issue mentions only about creation of UI and not about the implementation part, means a template is enough.
Can you show us the template here in the comments and everyone here can review it.

**Comment by aakankshaagr** [comment]
> I believe it is correct as the issue mentions only about creation of UI and not about the implementation part, means a template is enough.
> Can you show us the template here in the comments and everyone here can review it.

Yeah sure @Aaishpra.I have 2 templates in my mind.Other email templates are simple. So below one is the simple email which I thought of

![image](https://user-images.githubusercontent.com/71868375/109414507-380a5f00-79d9-11eb-900f-929892474a15.png)

 Shall I add a tick sign or party-popper image before congratulations at center and even increase text size?

![image](https://user-images.githubusercontent.com/71868375/109414630-ddbdce00-79d9-11eb-8fe3-66a878710ed9.png)

**Comment by Aaishpra** [comment]
@aakankshaagr the project you are working for is open source programs not mentorship system.
And i think the last line should be AnitaB.org along with the link to AnitaB website instead of systers open source.
And i like the above template better.

Tagging @isabelcosta and @codesankalp so that they can also give their input here.

**Comment by Aaishpra** [comment]
Now i see you were confused between mentorship system and Open Source Programs since start, as you mentioned about swagger UI in one of your comment.😅

No problem though, just need a little bit tweaks here and there and everything will be fine.

**Comment by aakankshaagr** [comment]
> Now i see you were confused between mentorship system and Open Source Programs since start, as you mentioned about swagger UI in one of your comment.😅
> 
> No problem though, just need a little bit tweaks here and there and everything will be fine.

Yeah I was confused between both 😅
thanks @Aaishpra , Now I got it. I will make changes as you have suggested.

**Comment by aakankshaagr** [comment]
I have made changes, as suggested by @Aaishpra.I, have attached the final drafts of both designs. @isabelcosta @Aaishpra @codesankalp Please review the designs.

Design 1 : 
![image](https://user-images.githubusercontent.com/71868375/109654045-be64a380-7b87-11eb-9838-23b4166ad358.png)

Design 2:
![image](https://user-images.githubusercontent.com/71868375/109653500-1bac2500-7b87-11eb-8b1b-94874d5004a1.png)
![image](https://user-images.githubusercontent.com/71868375/109653577-341c3f80-7b87-11eb-811a-bfc569650f30.png)

**Comment by codesankalp** [comment]
@Aaishpra I want to ask that is this issue is related to making a web-page for email confirmed UI to display on frontend or for making an email template that needs to be sent via mail after email is Confirmed?
And the draft made by @aakankshaagr looks good but looks like email templates.

**Comment by Aaishpra** [comment]
@codesankalp the issue description and issue heading seems very confusing now when I see it properly, for example, the issue headline suggests "Create email confirmed UI" which I think means creating an email template, but the issue description contains this line " I need to see an email confirmation message in osp once verification is done." which in my view means creating a design/UI for the webpage in OSP.
I am not sure but, @codesankalp do we have a template for the email confirmation in the project, If yes then the issue clearly means the creation of a webpage and if not then the issue suggests of creation of an email template.

Also, this issue is pretty old (open since the last GSoC period)so I am also confused with the requirements and needs for this issue.

**Comment by aakankshaagr** [comment]
I also had this doubt that @codesankalp has asked. According to the issue description, I think it will be displayed on the OSP after the account is verified. So, I think I have to create a webpage, not an email template.

**Comment by aakankshaagr** [comment]
Is this webpage okay @Aaishpra @codesankalp @isabelcosta?

![image](https://user-images.githubusercontent.com/71868375/109789051-edd1e980-7c35-11eb-9515-e3c2ed566482.png)
 I think I should remove "Welcome to AnitaB.Org" and in place of "To get started" ,`" For more information "` suits better.

**Comment by codesankalp** [comment]
@Aaishpra There is no email template for the email confirmation. 
But if we want to send it by email then this issue must be opened in the backend Repo.
There is so much confusion in this. Why don't we ask about it to @abha224 and @isabelcosta? They can better explain this.
